### PR TITLE
Replace `nuxtjs/svg` with `nuxtjs/svg-sprite`

### LINKED ISSUE #1849: Use `svg-sprite` module for loading all SVG files
## Problem

#1808 has introduced the Nuxt SVG sprite module to load the icons. Now we use two separate modules to load SVG images.

## Description

We should create a new component that wraps the SVG sprite module's `svg-icon` component to load the other SVG images. The new `VSvg` component should be created using `just frontend/run create:component`.

## Additional context

The jest svg transformers could probably also be removed after moving to `svg-sprite`, as noted [in this comment](https://github.com/WordPress/openverse/pull/1808#discussion_r1169530854).

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #1849 by @obulat

## Description

This PR replaces "nuxtjs/svg" library that does not have support for Nuxt 3 with "nuxtjs/svg-sprite". We already use it for icons.
To use it for the images of various sizes, we have to add a `viewBox` to the `VSvg` components - I couldn't find a better solution, unfortunately.

## Testing Instructions

All of the Visual Regression tests in the CI should still pass because there should be no visible change.
So, to test this PR you would need to just read through the code and check that the CI passes.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by sarayourfriend — APPROVED** [review]
This is great :tada:

File: frontend/src/components/VFourOhFour.vue [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
   <div class="error grid overflow-x-hidden">
-    <svg
+    <VSvg
       class="z-0 pointer-events-none col-start-1 row-start-1 -mx-[15%] fill-dark-charcoal opacity-5 lg:mx-15 lg:-mt-20"
-      xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 1320 569"
-      aria-hidden="true"
-      focusable="false"
-    >
-      <use :href="`${Oops}#oops`" />
```
Comment (AetherUnbound) [inline frontend/src/components/VFourOhFour.vue:10]: Should this href have been deleted?

File: frontend/src/components/VSvg/VSvg.vue [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <SvgIcon :name="`images/${name}`" aria-hidden="true" focusable="false" />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue"
+/**
+ * Displays the given SVG image.
+ */
+export default defineComponent({
+  name: "VIcon",
```
Comment (AetherUnbound) [inline frontend/src/components/VSvg/VSvg.vue:11]: Wait, is this `VIcon` or `VSvg`? 😮

**Review by AetherUnbound — COMMENTED** [review]
LGTM, but I have a few questions!

**Review by AetherUnbound — APPROVED** [review]
(don't want to block a merge though)

File: frontend/src/components/VFourOhFour.vue (hunk 1 above)
Comment (obulat) [reply frontend/src/components/VFourOhFour.vue:10]: Yes, the `<VSvg>` component handles finding the `href`s now, and we do not use the id of the inner SVG path. We only need to pass the name of the file to use it as an SVG.

File: frontend/src/components/VSvg/VSvg.vue (hunk 2 above)
Comment (obulat) [reply frontend/src/components/VSvg/VSvg.vue:11]: Great catch, left over from copy-pasting 😨

Comment (obulat) [inline frontend/src/components/VSvg/VSvg.vue:11]:
```suggestion
  name: "VSvg",
```

**Pull request merged**